include/psp2/pss.h

```diff
@@ -12,7 +12,16 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-
+  
+typedef struct ScePssHandle {
+  uint32_t unk0;
+  uint32_t unk1;
+  uint32_t filesz;
+  uint32_t unk3;
+} ScePssCryptoHandle;
+
+int pss_crypto_open_p(ScePssCryptoHandle *handle, char *path);
+char *pss_crypto_read_p(ScePssCryptoHandle *handle, int unk);
```

Comment:
Can you provide a known good value for `unk`?

Check other source files to see how you can document a function.

Reply:
It's a pointer to an int appears to always be 0x1 from what I've seen ?